build.gradle

```diff
@@ -797,9 +804,11 @@ jlink {
                 'org.bouncycastle.pqc.jcajce.provider.BouncyCastlePQCProvider'
         provides 'kong.unirest.core.json.JsonEngine' with 'kong.unirest.modules.gson.GsonEngine';
         provides 'ai.djl.repository.zoo.ZooProvider' with 'ai.djl.engine.rust.zoo.RsZooProvider',
-                'ai.djl.huggingface.zoo.HfZooProvider',
```

Comment:
I guess the PromptTemplateFactory from langchain needs to be added as well?

Reply:
This provides is for DJL - the `PromptTemplateFactory` is from langchain (`dev.langchain4j.spi.prompt.PromptTemplateFactory`).

Do we really need to adapt for langcahin if DJL has issues?

---

I tried the build of `jpackage` locally. And if I am not mistaken, it works.

Reply:
Ah, because of the `uses` update in module-info.java...

Reply:
`uses 'dev.langchain4j.spi.prompt.PromptTemplateFactory'` exist above. I don't see a statement regarding `provides`. Thus, it should be OK as is ^^

Reply:
Tested locally, works fine